src/plugins/menu_qrcode.py

```diff
@@ -1,20 +1,25 @@
 # -*- coding: utf-8 -*-
 """
+src/plugins/menu_qrcode.py
+==========================
+
 A menu plugin showing QR-Code for bitmessage address in modal dialog.
 """
 
+from pybitmessage.tr import _translate
+
 import urllib
 
 import qrcode
 from PyQt4 import QtGui, QtCore
 
-from pybitmessage.tr import _translate
-
 
 # http://stackoverflow.com/questions/20452486
-class Image(qrcode.image.base.BaseImage):
+class Image(qrcode.image.base.BaseImage):           # pylint: disable=abstract-method
```

Comment:
Where is the abstract method?

Reply:
Method 'save' is abstract in class 'BaseImage' but is not overridden